app/components/pages/Studio.vue.ts

```diff
@@ -76,14 +77,22 @@ export default class Studio extends Vue {
     const containerHeight = this.$root.$el.getBoundingClientRect().height;
 
     // This is the maximum height we can use
-    this.maxHeight = containerHeight - (this.performanceMode ? 200 : 400);
+    this.maxHeight = containerHeight - (this.performanceMode ? 200 : 1);
```

Comment:
not sure why we're leaving a 200px maxHeight for performance mode here?

Reply:
If you do enable performance mode, you probably want to see the button to disable performance mode?

Reply:
but could also just drag down the mini feed to do that too right?